### PR TITLE
use triggerClass

### PR DESCRIPTION
**Pull request by miguelcobain**

### CODE REVIEW

File: addon/templates/components/power-select/multiple.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{#basic-dropdown class=(readonly concatenatedClasses) dir=dir tabindex=(readonly tabindex) renderInPlace=renderInPlace matchTriggerWidth=true
-  disabled=disabled dropdownPosition=dropdownPosition triggerClass=(readonly triggerClass) dropdownClass=concatenatedDropdownClasses
+  disabled=disabled dropdownPosition=dropdownPosition triggerClass=concatenatedTriggerClasses dropdownClass=concatenatedDropdownClasses
```
Comment (cibernox) [inline addon/templates/components/power-select/multiple.hbs]: readonly

File: addon/templates/components/power-select/single.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{#basic-dropdown class=(readonly concatenatedClasses) dir=dir tabindex=(readonly tabindex) renderInPlace=renderInPlace matchTriggerWidth=true
-  disabled=disabled dropdownPosition=dropdownPosition triggerClass="ember-power-select-trigger" dropdownClass=concatenatedDropdownClasses
+  disabled=disabled dropdownPosition=dropdownPosition triggerClass=concatenatedTriggerClasses dropdownClass=concatenatedDropdownClasses
```
Comment (cibernox) [inline addon/templates/components/power-select/single.hbs]: readonly

**Comment by miguelcobain** [comment]
Couldn't add `readonly` on `opened` property. You have specific tests to assure it is mutable.
It is out of the scope of this PR to deal with that, IMO.

**Comment by cibernox** [comment]
Yes, `opened` is the only exception. It's a mutable property by design. I also commented that in the API reference.

**Pull request merged**